contracts/token/Fungi.sol

```diff
@@ -45,7 +47,12 @@ abstract contract Inscriptions is PoolCreatableErc20i {
     event OnSporesGrow(address indexed holder, SeedData seed_data);
     event OnSporesShrink(address indexed holder, SeedData seed_data);
 
-    constructor() PoolCreatableErc20i("Fungi", "FUNGI", msg.sender) {}
+    constructor() PoolCreatableErc20i("Fungi", "FUNGI", msg.sender) {
+        // Deployer must be added to holders list
+        _addHolder(msg.sender);
+        SeedData memory seed = SeedData(_TotalSupplyParam, 1); //We use the total supply param to properly set the seed, extra doesn't matter
+        _addSeedCount(msg.sender, seed.seed);

```

Comment:
any sends done prior to launch will also be 0 values and cause the holder drift issues. so its vital that launch is performed before sending tokens to any other address